utils/common.test.ts

```
@@ -0,0 +1,258 @@
+  });
+
+  describe('isNumber', () => {
+    it('should return true for numbers', () => {
+      expect(commonUtils.isNumber(123)).toBe(true);
+      expect(commonUtils.isNumber(0)).toBe(true);
+      expect(commonUtils.isNumber(-123)).toBe(true);
+      expect(commonUtils.isNumber(3.14)).toBe(true);
+    });
+    it('should return false for non-numbers', () => {
+      expect(commonUtils.isNumber('123')).toBe(false);
+      expect(commonUtils.isNumber(NaN)).toBe(false);
```

Biome (1.9.4)

[error] Use Number.NaN instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.NaN instead.


(lint/style/useNumberNamespace)